docs/utilities/custom_resources.md

```diff
@@ -34,7 +34,6 @@ To install this utility, add the following dependency to your project.
      dependencies {
         ...
         implementation 'software.amazon.lambda:powertools-cloudformation:1.10.0'
-        aspectpath 'software.amazon.lambda:powertools-cloudformation:1.10.0'
```

Comment:
Is this correct, it's `aspect` in other places.

Reply:
I have removed aspect from here since cloudformation does not do anything with aspects.. so its not needed